pkg/pillar/types/zedmanagertypes.go

```diff
@@ -159,6 +161,9 @@ type AppInstanceStatus struct {
 	State          SwState
 	MissingNetwork bool // If some Network UUID not found
 	MissingMemory  bool // Waiting for memory
+
+	EffectiveActivate bool //set here effective activate after profile check and apply
```

Comment:
I would also add a comment for AppInstanceConfig's Activate to point out that the EffectiveActivate in status must be used for the actual activation.